README.md

```diff
@@ -66,6 +66,19 @@ make up
 You can now connect to your local Airflow web console at
 `http://localhost:8000/`.
 
+### Testing Dev Changes
+
+*Note: This only works for `telemetry-batch-view` and `telemetry-streaming` jobs*
+
+A dev changes can be run by simply changing the `DEPLOY_TAG` environment variable
+to whichever upstream branch you've pushed your local changes to.
+
+Afterwards, you're going to need to rebuild: `make build`
+
+From there, you can either set the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` in the 
+Dockerfile and run `make up` to get a local UI and run from there, or you can follow the
+testing instructions above and use `make run`.
```

Comment:
Out of scope for this bug, but it kind of sucks that we're encouraging users to modify the docker files directly. For applications using docker-compose, I know we use .env files to specify environment-- not sure what the right solution is here.